build/vsts-ci.yml

```diff
@@ -226,21 +226,20 @@ phases:
       msbuildVersion: 15.0
     continueOnError: false
 
-  - task: NuGetCommand@2 
-    displayName: Publish Packages to VSTS Feed 
-    inputs:
-      command: push 
-      packagesToPush: $(Build.SourcesDirectory)/bin/packages/**/*.nupkg;!$(Build.SourcesDirectory)/bin/packages/**/*.symbols.nupkg 
-      nuGetFeedType: internal 
-      feedPublish: MachineLearning
-      
   # - task: MSBuild@1
```

Comment:
I wonder if I should delete this, since I believe this won't be used in the future?

Reply:
Yes, please remove.